src/main/java/com/onarandombox/MultiversePortals/listeners/MVPPlayerMoveListener.java

```diff
@@ -94,7 +94,13 @@ public void playerMove(PlayerMoveEvent event) {
                 return;
             }
 
-            MultiverseWorld world = this.plugin.getCore().getMVWorldManager().getMVWorld(d.getLocation(p).getWorld().getName());
+            Location destLocation = d.getLocation(p);
+            if (destLocation == null) {
+                Logging.fine("Unable to teleport player because destination is null!");
+                return;
+            }
+
+            MultiverseWorld world = this.plugin.getCore().getMVWorldManager().getMVWorld(destLocation.getWorld().getName());
             if (world == null) {
```

Comment:
maybe logging here as well? or it needs a separate PR since its not exactly about destination.

Reply:
I'll just put it in this PR.

Reply:
Alright cool!

Reply:
Okay, all done. I added the same condition in MVPPlayerListener too.